nightwatch/commands/mountVueComponent.js

```diff
@@ -196,6 +196,7 @@ module.exports = class Command {
       
       let element = mount(Component, {
         attachTo: document.getElementById('app'),
+        props: ${JSON.stringify(opts.props)},
```

Comment:
What happens if the `JSON.stringify` throws an Error?

Reply:
Nightwatch will propagate the error to the CLI